Libraries/PeriphDrivers/Source/I2C/i2c_me17.c

```diff
@@ -48,7 +48,10 @@
 #include "i2c_reva.h"
 
 /* **** Definitions **** */
+#define MXC_I2C_STD_MODE 100000
+#define MXC_I2C_FAST_SPEED 400000
 #define MXC_I2C_FASTPLUS_SPEED 1000000
+#define MXC_I2C_HS_MODE 3400000
```

Comment:
I think these suffixes should all match. Either all end in "_SPEED" or "_MODE", dealer's choice.

Reply:
+1 for `_SPEED`

Reply:
Renamed to HIGH_SPEED